ClientApp/package-lock.json

```diff
@@ -5,17 +5,17 @@
   "requires": true,
```

Comment:
The package.json file hasn't actually changed so I'm surprised these have all updated, I don't think they should have - could we revert the changes to this file and the package.json file please?